src/triage/experiments/validate.py

```diff
@@ -678,6 +678,13 @@ def _run(self, grid_config):
                 )
             )
         for classpath, parameter_config in grid_config.items():
+            if classpath == "sklearn.linear_model.LogisticRegression":
+                logging.warning(
+                    "sklearn.linear_model.LogisticRegression found in grid. "
+                    "This is unscaled and not well-suited for Triage experiments. "
+                    "Use triage.component.catwalk.estimators.classifiers.ScaledLogisticRegression "
+                    " instead"
```

Comment:
one too many spaces :stuck_out_tongue: 

```suggestion
                    "instead"
```